File: softwares/base16/output/crosh/colors/base16-papercolor-light.js
// Base16 PaperColor-Light
// Scheme: Nguyen Nguyen (http://github.com/NLKNguyen)

var color_scheme = {
        'base00': '#f3f3f3',
        'base01': '#f3f3f3',
        'base02': '#d0d0d0',
        'base03': '#949494',
        'base04': '#808080',
        'base05': '#4d4d4c',
        'base06': '#262626',
        'base07': '#262626',
        'base08': '#8959a8',
        'base09': '#d75f00',
        'base0A': '#4271ae',
        'base0B': '#718c00',
        'base0C': '#3e999f',
        'base0D': '#005f87',
        'base0E': '#d7005f',
        'base0F': '#df0000',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(77, 77, 76, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);

File: softwares/base16/output/pygments/pygments_base16/base16-papercolor-light.py
from pygments.style import Style
from pygments.token import (
    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
)


class Base16Style(Style):
    base00 = '#f3f3f3'
    base01 = '#f3f3f3'
    base02 = '#d0d0d0'
    base03 = '#949494'
    base04 = '#808080'
    base05 = '#4d4d4c'
    base06 = '#262626'
    base07 = '#262626'
    base08 = '#8959a8'
    base09 = '#d75f00'
    base0a = '#4271ae'
    base0b = '#718c00'
    base0c = '#3e999f'
    base0d = '#005f87'
    base0e = '#d7005f'
    base0f = '#df0000'

    default_style = ''

    background_color = base00
    highlight_color = base02

    styles = {
        Text: base05,
        Error: base08,  # .err

        Comment: base03,  # .c
        Comment.Preproc: base0f,  # .cp
        Comment.PreprocFile: base0b,  # .cpf

        Keyword: base0e,  # .k
        Keyword.Type: base08,  # .kt

        Name.Attribute: base0d,  # .na
        Name.Builtin: base0d,  # .nb
        Name.Builtin.Pseudo: base08,  # .bp
        Name.Class: base0d,  # .nc
        Name.Constant: base09,  # .no
        Name.Decorator: base09,  # .nd
        Name.Function: base0d,  # .nf
        Name.Namespace: base0d,  # .nn
        Name.Tag: base0e,  # .nt
        Name.Variable: base0d,  # .nv
        Name.Variable.Instance: base08,  # .vi

        Number: base09,  # .m

        Operator: base0c,  # .o
        Operator.Word: base0e,  # .ow

        Literal: base0b,  # .l

        String: base0b,  # .s
        String.Interpol: base0f,  # .si
        String.Regex: base0c,  # .sr
        String.Symbol: base09,  # .ss
    }


from string import capwords  # noqa: E402
Base16Style.__name__ = 'Base16{}Style'.format(
    capwords('papercolor-light', '-').replace('-', '')
)
globals()[Base16Style.__name__] = globals()['Base16Style']
del globals()['Base16Style']
del capwords
